helm-charts/cubestore/templates/pvc.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if and (not .Values.cloudStorage.gcp.bucket) (not .Values.cloudStorage.aws.bucket) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ printf "%s-remote-storage" (include "cubestore.fullname" .) }}
+  {{- if eq .Values.remoteDir.persistence.resourcePolicy "keep" }}
+  annotations:
+    helm.sh/resource-policy: keep
+  {{- end }}
+  labels: 
+    app.kubernetes.io/component: remote-storage
+    {{- include "cubestore.selectorLabels" . | nindent 4 }}
+  {{- if .Values.remoteDir.persistence.annotations }}
+  annotations: {{- toYaml .Values.remoteDir.persistence.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  accessModes:
+    - ReadWriteMany
```

Comment:
@lvauvillier it's always better not to use VPs with ReadWriteMany as well as other shared NFS-like storages vs just using S3 buckets. 
There are always problems with performance & permissions with NFS-like storages. 
All cloud providers have buckets & on prem can easily deploy smth like minio.